generators/client/templates/angular/src/main/webapp/app/account/password-reset/finish/password-reset-finish.component.html.ejs

```diff
@@ -21,11 +21,11 @@
         <div class="col-md-4">
             <h1 jhiTranslate="reset.finish.title">Reset password</h1>
 
-            <div class="alert alert-danger" jhiTranslate="reset.finish.messages.keymissing" *ngIf="keyMissing">
+            <div class="alert alert-danger" jhiTranslate="reset.finish.messages.keymissing" *ngIf="keyMissing === true">
                 <strong>The password reset key is missing.</strong>
             </div>
 
-            <div class="alert alert-warning" *ngIf="!success && !keyMissing">
+            <div class="alert alert-warning" *ngIf="!success && keyMissing === false">
```

Comment:
Why this condition is changed ? Shouldn't we check that keyMissing is false or null ?

Reply:
Because this message can be seen only after route query parameters are resolved. So `keyMissing` had 3 values, `null` meant **not initialized**.  
As this code was confusing, I removed `keyMissing` and added `initialized` variable now.